### ISSUE TITLE
ohm-js cli : add --output argument to define the output folder

### ISSUE THREAD

**Issue by stondini** [ohmjs/ohm#385]
Hello,

It may be useful for Typescript developers to have the ability to generate the Javascript + Typescript definition files in a specific folder in order to avoid them in `src` folder (which is the current output folder).

For instance, the following command generates the files at the same place as the ohm grammar source files.
`ohm generateBundles --withTypes src/ohm/*.ohm`

It could be great to have an additional argument to specify the output folder:
`ohm generateBundles --withTypes src/ohm/*.ohm --output dist/ohm`

What do you think ?

Thank you.
Stephane

**Comment by pdubroy** [comment]
Agreed! My instinct would be to make this behave the same way as [TypeScript's `outDir` arg](https://www.typescriptlang.org/tsconfig#outDir), and preserve the directory structure of the original files. Does this seem like the right choice?

**Comment by stondini** [comment]
Yes it sounds good.
It could be the default behaviour and stil keep `--output` argument to override it in case of special situations.
